tests/test_warc_metadata_sidecar.py

```diff
@@ -0,0 +1,65 @@
+import io
+import os
+from logging import INFO
+from unittest.mock import patch
+
+from warcio.archiveiterator import ArchiveIterator
+from warcio.recordloader import ArcWarcRecord
+
+import warc_metadata_sidecar as sidecar
+
+
+class Test_Warc_Metadata_Sidecar:
+    def get_record(self):
+        test_file = os.path.join(os.path.dirname(__file__), 'warc.warc')
+        with open(test_file, 'rb') as stream:
+            for record in ArchiveIterator(stream):
+                url = record.rec_headers.get_header('WARC-Target-URI')
+                rawPayload = io.BytesIO(record.raw_stream.read())
+                if record.http_headers:
+                    recordCopy = ArcWarcRecord(
+                        record.format, record.rec_type,
+                        record.rec_headers, rawPayload,
+                        record.http_headers, record.content_type,
+                        record.length
+                        )
+                    decodedPayload = recordCopy.content_stream().read()
+                    rawPayload.seek(0)
+                else:
+                    decodedPayload = rawPayload
+            return(url, rawPayload, decodedPayload)
```

Comment:
1. I see in the test file this uses, there is only one record, so this works ok. However, the code reads odd because if the WARC had more than one record, this code indicates to loop over all the records, do work for them all, then just return the last one. See fixture sample code for another option.
2. If retaining the `return` line, probably add a space after `return`.